genshin/client/components/chronicle/genshin.py

```diff
@@ -107,6 +107,22 @@ async def get_genshin_spiral_abyss(
 
         return models.SpiralAbyss(**data)
 
+    async def get_imaginarium_theater(
+        self,
+        uid: int,
+        *,
+        previous: bool = False,
+        lang: typing.Optional[str] = None,
+    ) -> models.ImgTheater:
+        """Get Genshin Impact imaginarium theater runs."""
+        payload = {
+            "schedule_type": 2 if previous else 1,  # There's 1 season for now but I assume it works like this
+            "need_detail": "true",
```

Comment:
`need_detail` can be `False` for querying other uids.

Reply:
Uhh how do I know if the user is querying other UIDs?
I saw [this](https://github.com/seriaati/genshin.py/blob/0a9558977cbe7300d726b6a64707286596163f5d/genshin/client/components/chronicle/starrail.py#L127) in starrail so I assumed we're doing the same here.

Reply:
starrail can query other user's detail , genshin cannot. Just take `need_detail` as an parameter to the function.